docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: "3"
+
+services:
+  db:
+    image: "postgres"
+    restart: unless-stopped
+    networks:
+      - "mylocal"
+    env_file:
+      - ".env"
+    volumes:
+      - ./db-data:/var/lib/postgresql/data
```

Comment:
The volume folder is in .gitignore. How this is intended to work after I clone the repo and try to start the service?

Reply:
It create folder db-data automaticaly

Reply:
Ok, and why we need this volume now?